test/e2e/framework/networking_utils.go

```diff
@@ -292,7 +292,7 @@ func (config *NetworkingTestConfig) DialFromNode(protocol, targetIP string, targ
 	if protocol == "udp" {
 		// TODO: It would be enough to pass 1s+epsilon to timeout, but unfortunately
 		// busybox timeout doesn't support non-integer values.
-		cmd = fmt.Sprintf("echo 'hostName' | nc -w 1 -u %s %d", targetIP, targetPort)
+		cmd = fmt.Sprintf("echo hostName | nc -w 1 -u %s %d", targetIP, targetPort)
```

Comment:
spaces are not permitted in hostnames for both Windows and Linux so the quotes change seems fine.
in terms of `nc` there is no bundled netcat for Windows or in popular packages like `msys`, so i don't see how this helps the overall Windows situation with `DialFromNode()`, unless you downloaded a tool from somewhere. :)

Reply:
Indeed, we are including the nc executable in the hostexec image: https://github.com/kubernetes-sigs/windows-testing/blob/f96cb4cc6d71cc5b9782306ed89cd93d1363ccd8/images/hostexec/Dockerfile#L17

Reply:
ack